stix_shifter_modules/cybereason/stix_transmission/api_client.py

```diff
@@ -32,7 +32,7 @@ def ping_box(self):
         query = '{"queryPath": [{"requestedType": "Connection", "filters": [], "isResult": true}],' \
                 '"totalResultLimit": 1, "perGroupLimit": 1,"templateContext": "SPECIFIC"}'
         headers['Cookie'] = self.session_log_in()
-        return self.client.call_api(self.QUERY_ENDPOINT, 'POST', headers=headers, data=query)
+        return self.client.call_api(self.QUERY_ENDPOINT, 'HEAD', headers=headers, data=query)
```

Comment:
in this case, i think we dont even need to make any QUERY_ENDPOINT call.  if `self.session_log_in()` is success then we can set ping successful.

Reply:
there is no way to know if it is successful with underlying call, it always return something like:`JSESSIONID=281AD9E056714688EA6F3423AC21E`